src/rgw/rgw_sal_motr.cc

```diff
@@ -176,6 +176,35 @@ inline std::string get_bucket_name(const std::string& tenant,  const std::string
     return bucket;
 }
 
+int static update_bucket_stats(const DoutPrefixProvider *dpp, MotrStore *store,
+                               std::string owner, std::string bucket_name,
+                               uint64_t size, uint64_t actual_size,
```

Comment:
Please add a comment stating what `size` and `actual_size` exactly represents.

Reply:
sure will address with the next PR regarding Stats.